### DISCUSSION TITLE
carga de imagenes en cloudinary no funciona en produccion

### DISCUSSION THREAD

**Discussion by jrccdevs** [vercel/community#5330] [Help]
### Summary

tengo una aplicacion creada en node y react y estoy utilizando cloudinary para las imagenes la carga funciona correctamente en localhost pero no funciona cuando esta en vercel en produccion solo con las imagenes  

Por favor si me pueden ayudar

### Steps to Reproduce

import { v2 as cloudinary } from "cloudinary";
import { API_KEY, API_SECRET, CLOUD_NAME } from "../config.js";

cloudinary.config({
  cloud_name: CLOUD_NAME,
  api_key: API_KEY,
  api_secret: API_SECRET,
  secure: true
  
});
 export const uploadControl = async (filePath) => {
    try {
    return await cloudinary.uploader.upload(filePath, {
    
    folder: "controlAlfaSA",
    });
  } catch (error) {
    console.error("Error al cargar la imagen:", error);
    throw error;  // Re-lanza el error para que pueda ser manejado más arriba
  }
};

//AQUI ESTA MI CODIGO DEL CONTROLLERS

import { uploadControl } from "../libs/cloudinary.js";
import { pool } from "../db.js";
import fs from 'fs-extra';

//creando imagenes 

export const createImagenes = async (req, res) => {

    try {
    const { nombre,
      estado,
      categoria,
      accion,
      tipo,
       } = req.body;
  
    let image;
    if (req.files?.image) {
      const result = await uploadControl(req.files.image.tempFilePath);
     
      await fs.remove(req.files.image.tempFilePath);
      image = result.secure_url
        //public_id: result.public_id,
     // };
    }
    //const {imagen} = image;
    const [resultado] = await pool.query(
      "INSERT INTO tblimagenes(nombre, image, categoria, estado, accion, tipo) VALUES (?, ?, ?, ?, ?, ?)",
      [nombre,
        image,
      categoria,
      estado,
      accion,
      tipo,
        ]
    );
    console.log(resultado)
  
  
    res.json({
      id: resultado.insertId,
      nombre,
      image,
      categoria,
      estado,
      accion,
      tipo,
    });
     } catch (error) {
       console.log(error)
       return res.status(500).json({ message: error.message });
       console.log(error)
     }
  };

**Comment by fiufiu25** [comment]
podistes solucionarlo
